main_test.go

```diff
@@ -77,23 +77,33 @@ func TestRunWithOptions(t *testing.T) {
 		wants   []string
 	}{
 		{
-			desc:   "No thrift file, fail to get method spec",
+			desc: "No thrift file, fail to get method spec",
+			opts: Options{
+				TOpts: TransportOptions{
+					ServiceName: "foo",
+					Peers:       []string{"1.1.1.1"},
```

Comment:
Should we make these `127.0.0.1`, so if for some reason we end up in a codepath that resolves this we don't block on trying to access the internet?

Reply:
:+1: 

updated this test. there are a lot of other references, might do a follow up for those.